uoft-tapp/tapp
# [Database] TAPP-130: Create new seed data & add new ER diagram

Please see issue #130 for more details.

To test this branch:
1. Run `docker-compose up --build`
2. Run `docker-compose exec rake db:migrate`
3. Run `docker-compose exec rake db:seed`

The last command should print out the list of tables that has been updated/inserted with new data. Furthermore, you should be able to re-run the last command several times without causing issues with the migration. The database should also update if you change the corresponding JSON files in the `/api/db/seed` folder.
